app.rb

```diff
@@ -37,6 +38,27 @@ def config
       @config ||= Config.new
     end
 
+    get '/attempt-events' do
```

Comment:
This probably doesn't belong here long-term and should be tidied up if it's going to stay. But this is a POC for now.